Formula/pulseaudio.rb

```diff
@@ -39,26 +39,69 @@ class Pulseaudio < Formula
   uses_from_macos "expat"
   uses_from_macos "m4"
 
+  on_linux do
+    depends_on "dbus"
+    depends_on "glib"
+    depends_on "libcap"
+
+    resource "XML::Parser" do
+      url "https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-2.44.tar.gz"
+      sha256 "1ae9d07ee9c35326b3d9aad56eae71a6730a73a116b9fe9e8a4758b7cc033216"
+    end
+  end
+
   def install
+    on_linux do
+      ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+      resource("XML::Parser").stage do
+        system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+        system "make", "PERL5LIB=#{ENV["PERL5LIB"]}", "CC=#{ENV.cc}"
+        system "make", "install"
+      end
```

Comment:
I think the Perl resource is a build-time dependency used in local `xmltoman` script to create man page, so it may be better to use `buildpath` rather than installing into `libexec` (and storing inside bottle).

Reply:
How do you do that? We have been using the current syntax in a few formulae now, so maybe this could be improved for all of them in a second PR?